manifests/params.pp

```diff
@@ -22,13 +22,19 @@
 #   The oauth key for Candlepin
 # @param candlepin_oauth_secret
 #   The oauth secret for Candlepin
+# @param scl_postgresql
+#   The scl name for PostgreSQL
+# @param postgresql_evr_package
+#   The contextual package name for the PostgreSQL EVR extension
 class katello::params (
   Stdlib::HTTPSUrl $pulp_url = "https://${facts['fqdn']}/pulp/api/v2/",
   Stdlib::HTTPSUrl $crane_url = "https://${facts['fqdn']}:5000",
   Stdlib::Host $qpid_hostname = 'localhost',
   String[1] $candlepin_oauth_key = $katello::globals::candlepin_oauth_key,
   String[1] $candlepin_oauth_secret = $katello::globals::candlepin_oauth_secret,
   Stdlib::Host $candlepin_host = 'localhost',
-  Stdlib::HTTPSUrl $candlepin_url = "https://${candlepin_host}:8443/candlepin"
+  Stdlib::HTTPSUrl $candlepin_url = "https://${candlepin_host}:8443/candlepin",
+  String[1] $scl_postgresql = 'rh-postgresql12',
+  String[1] $postgresql_evr_package = 'postgresql-evr'
```

Comment:
What you can do is the following in `globals.pp`
```puppet
if $facts['os']['release']['major'] == '7' {
  $postgresql_evr_package = 'rh-postgresql12-postgresql-evr'
} else {
  $postgresql_evr_package = 'postgresql-evr'
}
```

Then:
```suggestion
  String[1] $postgresql_evr_package = $katello::globals::postgresql_evr_package,
```

This gives you sane defaults and ready for EL8 but allows you to override the package name if you need to and you want to experiment (via custom-hiera for example). If we change the SCL, we'd do a major version bump and change the hardcoded default.